resources/configurationFiles/analysisRNAseqGRCh38.xml

```diff
@@ -7,47 +7,50 @@
-        <cvalue name='hg38DatabaseDirectory' value='${hg38BaseDirectory}/databases' type='string'/>
-        <cvalue name='hg38IndexDirectory' value='${hg38BaseDirectory}/indexes' type='string'/>
+        <cvalue name='hg38BaseDirectory' value='/omics/odcf/reference_data/by-species/${SPECIES}/${REF_GENOME}/${REF_GENOME_VERSION}' type='path' />
 
-        <cvalue name='databaseDirectory' value='${hg38DatabaseDirectory}' type='string'/>
-        <cvalue name='indexDirectory' value='${hg38IndexDirectory}' type='string'/>
+        <cvalue name='hg38BaseDirectory_legacy' value='/omics/odcf/reference_data/legacy/ngs_share/assemblies/hg_GRCh38/' type='path'/>
+        <cvalue name='hg38DatabaseDirectory_legacy' value='${hg38BaseDirectory_legacy}/databases' type='string'/>
 
-        <!--Databases and index-->
-        <cvalue name='GENOME_FA' value='${hg38BaseDirectory}/sequence/GRCh38_decoy_ebv_phiX_alt_hla_chr.fa' type='string'/>
-        <cvalue name='GENOME_GATK_INDEX' value='${hg38IndexDirectory}/bwa/bwa07_GRCh38_decoy_ebv_phiX_alt_hla_chr/GRCh38_decoy_ebv_phiX_alt_hla_chr.fa' type='string'/>
+        <cvalue name='databaseDirectory' value='${hg38DatabaseDirectory_legacy}' type='string'/>
+        <cvalue name='indexDirectory' value='${hg38BaseDirectory}' type='string'/>
 
-        <cvalue name='GENE_MODELS' value='${hg38DatabaseDirectory}/gencode/gencode31/hg38_gencode.v31.annotation.gtf' type='string'/>
-        <cvalue name='GENE_MODELS_NOGENE' value='${hg38DatabaseDirectory}/gencode/gencode31/hg38_gencode.v31.annotation.nogene.gtf' type='string' 
+        <cvalue name='GENOME_FA' value='${hg38BaseDirectory}/genome.fa' type='string'/>
+        <cvalue name='GENOME_GATK_INDEX' value='${GENOME_FA}' type='string'/>
+
+        <cvalue name='GENE_MODELS' value='${databaseDirectory}/gencode/${REF_GENOME_VERSION}/${GENCODE_VERSION}/annotation.gtf' type='string'/>
+        <cvalue name='GENE_MODELS_NOGENE' value='${databaseDirectory}/gencode/${REF_GENOME_VERSION}/${GENCODE_VERSION}/annotation.nogene.gtf' type='string' 
                 description="Like GENE_MODELS, but without 'gene' features. Required for gencode update for hg38. Defaults (for hg19): $GENE_MODELS."/>
 
-        <cvalue name='GENE_MODELS_EXCLUDE' value='${hg38DatabaseDirectory}/gencode/gencode31/hg38_gencode.v31.annotation.chrXYMT.rRNA.gtf' type='string'
+        <cvalue name='GENE_MODELS_EXCLUDE' value='${databaseDirectory}/gencode/${REF_GENOME_VERSION}/${GENCODE_VERSION}/annotation.chrXYMT.rRNA.gtf' type='string'
                 description="Other than mitochondrial genes, there were no genes annotated as tRNA, so have removed it from the filename."/>
-        <cvalue name='GENE_MODELS_DEXSEQ' value='${hg38DatabaseDirectory}/gencode/gencode31/hg38_gencode.v31.annotation.dexseq.gff' type='string'/>
+        <cvalue name='GENE_MODELS_DEXSEQ' value='${databaseDirectory}/gencode/${REF_GENOME_VERSION}/${GENCODE_VERSION}/annotation.dexseq.gff' type='string'/>
```

Comment:
Have removed the concatenated transcript IDs to avoid the memory issue related to feature names. [Issue in google groups](https://groups.google.com/g/subread/c/zJNSjxgsOrw/m/kied14tKAQAJ).

Commenting out [this line]( https://github.com/vivekbhr/Subread_to_DEXSeq/blob/23ccd2ff4ff605622d3f3575130b31b6761a0990/dexseq_prepare_annotation2.py#L126) during annotation preparation works.

Reply:
You said you have a script for preparing all the additional annotation files (as extension to refmake). Does that also include this modification? Do you think it would make sense to include that script in this workflow, here, to document its input?

Reply:
That's a good idea.. I have documented these scripts now.